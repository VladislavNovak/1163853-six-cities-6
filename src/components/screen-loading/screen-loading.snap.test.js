import React from 'react';
import {render} from '@testing-library/react';

import {ScreenLoading} from '..';

test(`Should ScreenLoading render correctly`, () => {
  const {container} = render(<ScreenLoading />);
  expect(container).toMatchSnapshot();
});
