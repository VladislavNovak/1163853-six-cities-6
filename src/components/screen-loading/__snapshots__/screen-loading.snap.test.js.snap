// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should ScreenLoading render correctly 1`] = `
<div>
  <div
    class="sc-fubCfw jVEaPA"
  >
    <p
      style="font-family: monospace; text-align: center; font-size: 28px; color: rgb(30, 144, 255); line-height: 26px; font-weight: 700;"
    >
      ...LOADING...
    </p>
    <div
      class="sc-pFZIQ edStyG"
    />
  </div>
</div>
`;
